nixos/modules/services/audio/navidrome.nix

```diff
@@ -28,10 +28,17 @@ in {
         '';
       };
 
+      openPort = mkOption {
```

Comment:
I think this option is most often named `openFirewall`.

Reply:
I copied the name from an existing module, but happy to change.

Reply:
Commit message still says "openPort".